--- /Users/sebastian/workspace/textploration/txpn/txpn-core/lib/txpn-core/dataModel.js
+++ /Users/sebastian/workspace/textploration/txpn/txpn-core/lib/txpn-core.js
@@ -8,5 +8,102 @@
  else
    root["TxpnCore"] = factory();
 })(this, function() {
-return webpackJsonpTxpnCore([1],[],[1]);
+return webpackJsonpTxpnCore([0],{
+
+/***/ 4:
+/***/ (function(module, exports, __webpack_require__) {
+
+"use strict";
+
+
+Object.defineProperty(exports, "__esModule", {
+  value: true
+});
+exports.SampleData = exports.dataModel = undefined;
+
+var _dataModel = __webpack_require__(1);
+
+var dataModel = _interopRequireWildcard(_dataModel);
+
+var _SampleData = __webpack_require__(8);
+
+var _SampleData2 = _interopRequireDefault(_SampleData);
+
+function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
+
+function _interopRequireWildcard(obj) { if (obj && obj.__esModule) { return obj; } else { var newObj = {}; if (obj != null) { for (var key in obj) { if (Object.prototype.hasOwnProperty.call(obj, key)) newObj[key] = obj[key]; } } newObj.default = obj; return newObj; } }
+
+exports.dataModel = dataModel;
+exports.SampleData = _SampleData2.default;
+
+/***/ }),
+
+/***/ 8:
+/***/ (function(module, exports, __webpack_require__) {
+
+"use strict";
+
+
+Object.defineProperty(exports, "__esModule", {
+  value: true
+});
+
+var _dataModel = __webpack_require__(1);
+
+var entranceHall = new _dataModel.Room({
+  name: 'Entrance Hall',
+  description: 'The spooky spooks.',
+  doors: []
+});
+
+
+var checkoutCounter = new _dataModel.Room({
+  name: 'Checkout Counter',
+  description: "Don't stay here too long...",
+  doors: []
+});
+
+var d2CheckoutCounter = new _dataModel.Door({
+  id: '1,1',
+  name: 'Library Front Door',
+  number: 1,
+  destination: checkoutCounter
+});
+
+var d2EntranceHall = new _dataModel.Door({
+  id: '1,1',
+  name: 'Library Front Door',
+  number: 1,
+  destination: entranceHall
+});
+
+checkoutCounter.doors.push(d2EntranceHall);
+entranceHall.doors.push(d2CheckoutCounter);
+
+var nvpl = new _dataModel.Region({
+  name: 'Nightvale Public Library',
+  rooms: [entranceHall, checkoutCounter]
+});
+
+var worlds = [new _dataModel.World({
+  id: 1,
+  name: 'Nightvale',
+  regions: [nvpl]
+}), new _dataModel.World({
+  id: 2,
+  name: 'Desert Bluffs'
+})];
+
+var explorer = new _dataModel.Explorer('Jimbob');
+
+var SampleData = {
+  worlds: worlds,
+  explorer: explorer
+};
+
+exports.default = SampleData;
+
+/***/ })
+
+},[4]);
 });
